### ISSUE TITLE
Emtpy cloned dataset from OSF

### ISSUE THREAD

**Issue by zhao-cy** [datalad/datalad-osf#160]
Hi there,

I hope to push a datalad dataset onto OSF so that I can clone it elsewhere on another computer. I first refered the [Quick Start](http://docs.datalad.org/projects/osf/en/latest/settingup.html) as it says we can clone the dataset later. There was no warning when creating OSF sibling or pushing to OSF. However the dataset I cloned from OSF is empty. Meanwhile, I tried a more complicated command (with more arguments) and it succeeded. In details:

When creating OSF sibling + pushing - commands that led to failure:
```
datalad create-sibling-osf --title data4babs_rawBIDS_multises -s osf
# # printed messages:
# create-sibling-osf(ok): https://osf.io/my5b7/
# [INFO   ] Configure additional publication dependency on "osf-storage" 
# configure-sibling(ok): . (sibling)

datalad push --to osf
# all okay, no error, no warning
```

Then on the OSF webpage, I change the OSF project to public. When cloning on another computer:
```
datalad clone osf://my5b7
# # printed messages:
# [WARNING] You appear to have cloned an empty repository.                                     
# [WARNING] Cloned /path/to/my/local/folder/my5b7 but could not find a branch with commits 
install(ok): /path/to/my/local/folder/my5b7 (dataset)

ls -a my5b7/
# printed messages:
# .    ..   .git
```
Basically the cloned dataset from OSF is empty, without files or folders.

Very interestingly, I tried using a more complicated datalad command (ref: [here](http://docs.datalad.org/projects/osf/en/latest/tutorial/cloneosfdata.html)), and it succeeded:
```
datalad create-sibling-osf --title data4babs_testout -s osf2 --category data --tag reproducibility --public
# # printed messages:
# create-sibling-osf(ok): https://osf.io/fhm8b/
# [INFO   ] Configure additional publication dependency on "osf2-storage" 
# configure-sibling(ok): . (sibling)
```

I'm wondering if there is anything missing in the simplest version of the `datalad create-sibling-osf` command I first tried?

Versions: 
  - datalad 0.17.2
  - datalad_osf 0.2.3.1

Thank you so much in advance!
Best,
Chenying

**Comment by mih** [comment]
The prominent difference I notice is that you have `--public` in the call that leads to a working sibling. I wonder if setting the project to public via the web UI is not 100% equivalent. I need to investigate that.

**Comment by yarikoptic** [comment]
FWIW I have tried to clone another fresh (not mine) but random OSF dataset and got similar result
```shell
❯ git clone osf://n6tma/
Cloning into 'n6tma'...
warning: You appear to have cloned an empty repository.
❯ git -C n6tma status
On branch master

No commits yet

nothing to commit (create/copy files and use "git add" to track)

```

**Comment by adswa** [comment]
But that particular dataset is an annex-only storage sibling, right, @yarikoptic? If so, there can't be any commits on there. I would assume that that particular repo has a sibling on GitHub somewhere. But its an issue that the autogenerated project description on OSF gives instructions for cloning - it makes it sound like that would work
